app/models/digital_service_account.rb

```diff
@@ -13,24 +13,24 @@ class DigitalServiceAccount < ApplicationRecord
   belongs_to :organization, optional: true
 
   validates :name, presence: true
-  validates :name, uniqueness: { scope: :account }
-  validates :account, presence: true
```

Comment:
Any reason not to drop the `account` column in the db?

Reply:
can do that as a follow-up

Reply:
added